.github/actions/setup-spack-stack/action.yaml

```diff
@@ -280,27 +285,29 @@ runs:
     if: ${{ inputs.concretize_only == 'false' }}
     shell: bash
     env:
-      OMP_NUM_THREADS: 4
+      OMP_NUM_THREADS: 2
     run: |
       cd ${{ inputs.path }}
       source setup.sh
       spack env activate ${{ inputs.path }}/envs/${{ inputs.name }}
-      # Speed up installation by building in parallel
-      for i in {1..2}; do
-        nohup spack install --fail-fast >> spack_install.log 2>&1 &
-      done
-      #
-      while ps -ef | grep -ve 'grep' | grep 'spack install'; do
-        echo "Still running"
-        tail -n 10 spack_install.log
-        sleep 60
-      done
+      ## Note: This is unreliable ...
+      ## Speed up installation by building in parallel
```

Comment:
What makes it unreliable? I used this technique myself after seeing it here and it worked fine.